### ISSUE TITLE
Passing template scope variables to VM property

### ISSUE THREAD

**Issue by daveeel** [ManuelDeLeon/viewmodel#132]
I got some jade like this:
~~~
.svg-thumbnail(id="{{ this.playId }}" $b="html: svgHTML( this.playId )") 
~~~
That's not working with error: TypeError: Cannot use 'in' operator to search for 'false' in undefined
and using playId alone. error is something like VM don't have "playId" property.

Just want to pass that playId in template to the svgHTML VM function to return some html depending on different id. Seems obvious the 2 "this" referring to different scopes. 

How to do what I need?

**Comment by daveeel** [comment]
I could fall back to call standard template helper like:
~~~
  .svg-thumbnail(id="svg-{{ this.playId }}")  {{ svgHTMLHelper this.playId }}
~~~
where the helper uses d3's raw power on DOM. But using VM binding should be cleaner.

**Comment by ManuelDeLeon** [comment]
I need more information. Is that inside a loop? Show me the code.

**Comment by daveeel** [comment]
Yes, it's inside a loop. 

~~~
each chainData
  .svg-thumbnail(id="{{ this.playId }}" $b="html: svgHTML( this.playId )") 
~~~

Where chainData is a Template helper returning a collection with .playId property. It's fine at the id = ... part, just can't get the binding working.

**Comment by ManuelDeLeon** [comment]
Please make a repro because I'm just guessing here.

**Comment by daveeel** [comment]
ok, will try to isolate and related code later and let you know. thanks.

**Comment by daveeel** [comment]
I just made a simple project and found that the following works as expected:
~~~
each chainData
  .svg-thumbnail(id="{{ this.playId }}" $b="html: svgHTML(this.playId)") 
~~~
The difference is that there can't be a leading space before the parameter, 
e.g. "svgHTML( this.playId)" and "svgHTML( this.playId ) will cause the error
but "svgHTML(this.playId)" and "svgHTML(this.playId ) are both fine 

So maybe it's a parsing issue?

**Comment by ManuelDeLeon** [comment]
Interesting. It should work with the space. I'll test/fix it.

**Comment by ManuelDeLeon** [comment]
This has been fixed on 2.5.1

Thanks for the help.
